docs/DEVELOPING.md

```diff
@@ -343,6 +343,111 @@ $ make ftest NAME=azure_blob_storage
 
 ℹ️ The e2e test uses default values defined in [configure Azure Blob Storage connector](#configure-azure-blob-storage-connector)
 
+
+## PostgreSQL Connector
+
+The [Elastic PostgreSQL connector](https://github.com/elastic/connectors-python/blob/8.7/connectors/sources/postgresql.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+⚠️ _Currently, this connector is available in **beta** starting in 8.7_.
+Features in beta are subject to change and are not covered by the service level agreement (SLA) of features that have reached general availability (GA).
+
+Elastic versions 8.6.0+ are compatible with Elastic connector frameworks. Your deployment must include the Elasticsearch, Kibana, and Enterprise Search services.
+
+### Setup and basic usage
+
+Complete the following steps to deploy the connector:
+
+1. [Gather PostgreSQL details](#gather-postgresql-details)
+2. [Configure PostgreSQL connector](#configure-postgresql-connector)
+
+#### Gather PostgreSQL details
+
+Collect the information that is required to connect to your PostgreSQL server:
+
+- The server host address where the PostgreSQL is hosted.
+- The port where the PostgreSQL is hosted.
+- The username the connector will use to log in to the PostgreSQL.
+- The password the connector will use to log in to the PostgreSQL.
+- The database name where the connector will query data.
+- SSL certificate if you want to establish secured connections.
+
+ℹ️ User needs to set `track_commit_timestamp` to `on` using `ALTER SYSTEM SET track_commit_timestamp = on;` query to fetch last updated time.
+
+#### Configure PostgreSQL connector
+
+The following configuration fields need to be provided for setting up the connector:
+
+##### `host`
+
+The server host address where the PostgreSQL is hosted. Default value is `127.0.0.1`. Examples:
+
+  - `192.158.1.38`
+  - `demo.instance.demo-region.demo.service.com`
+
+##### `port`
+
+The port where the PostgreSQL is hosted. Default value is `9090`. Examples:
+
+  - `5432`
+  - `9090`
+
+##### `username`
+
+The username of the account for PostgreSQL. Default value is `admin`.
+
+##### `password`
+
+The password of the account to be used for the PostgreSQL. Default value is `Password_123`.
+
+##### `database`
+
+Name of the PostgreSQL database. Default value is `xe`. Examples:
```

Comment:
This is a copy and paste error. There's no default `xe` Postgres database, that's the default for Oracle Express Edition 😄 My 2 cents is to have no default for this field but make it required to be set.

Reply:
We have a configuration list in generic_database.py and `database` parameter is common for all 3 dialects (Postgresql, Oracle, and MS-SQL). To run E2E tests, we need to set one default database in the generic_database file. So, that's why it's common in all documentation PR.

Reply:
It probably makes sense to call out that the default value exists for E2E testing purposes.

Reply:
Sure added note [here](https://github.com/elastic/connectors-python/pull/549/files#diff-f8ff34865ce88864196f2848aebde961f27a4c1e8ce77313a4254dd948d32cfbR467)